keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -379,14 +383,39 @@ command or function to work in the same way in another tool. This can
 further enhance the usability and adoption of different cluster
 manager.
 
-### Notes/Constraints/Caveats (Optional)
+### Notes/Constraints/Caveats
+#### What's the relationship between the ClusterProfile API and Cluster Inventory?
+The ClusterProfile API represents a single member cluster in a cluster inventory.
+
+#### What's the relationship between a cluster inventory and clusterSet?
+A cluster inventory may or may not represent a ClusterSet. A cluster inventory is considered a [clusterSet](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api#terminology)
+if all its member clusters adhere to the [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md) principle.
+In other terms, a clusterSet is a special type of cluster inventory while the reverse is not true.
+Note that a cluster can only be in one ClusterSet while there is not such restriction for a cluster inventory.
+
+#### How should the API be consumed?
+We recommend that all ClusterProfile objects within the same cluster inventory reside on 
+a dedicated hub Kubernetes cluster. This approach allows consumers to have a single integration 
```

Comment:
Have we defined "hub Kubernetes cluster" somewhere?

Reply:
Good question. Not yet, maybe we don't need to define what is the hub Kubernetes cluster.

Given the ClusterProfile is created by the `cluster manager`, so I guess here we can say the cluster manager organizes the ClusterProfile.